drush/encrypt.drush.inc

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/**
+ * @file
+ * Drush integration for Encrypt module.
+ */
+
+use Drupal\encrypt\Entity\EncryptionProfile;
+
+/**
+ * Implements hook_drush_command().
+ */
+function encrypt_drush_command() {
+  $items = array();
+
+  $items['encrypt'] = array(
```

Comment:
I think we should have a "test profile" one that runs through a test with a random string